python/pyspark/ml/torch/distributor.py

```diff
@@ -41,6 +41,7 @@
 )
 
 from pyspark import cloudpickle
+from pyspark.pandas.missing import frame
```

Comment:
This seems to have been accidentally added; remove this

Reply:
doen